Joysticks/mfjoystick.schema.json

```diff
@@ -27,11 +27,11 @@
         "type": "object",
         "required": [ "Id", "Label", "Type"],
         "properties": {
-          "Name": {
+          "Id": {
             "$id": "#root/Inputs/items/Id",
             "title": "Id",
             "description": "The input's unique identifier on the joystick. Required.",
-            "type": "string"
+            "type": "integer"
```

Comment:
I am not a big friend of integer ids. This should better be string for more flexibility.

Reply:
The code uses ints and always has, this was just changing the schema to match what the code already did.